libhxcfe/sources/floppy_loader.c

```diff
@@ -1,6 +1,6 @@
 /*
 //
-// Copyright (C) 2006-2023 Jean-Fran�ois DEL NERO
+// Copyright (C) 2006-2023 Jean-François DEL NERO

```

Comment:
Encoding problems after I edited / saved the files. Maybe your original source wasn't UTF-8 or there was an issue in the SF -> Github migration ?